bucket/networkmanager.json

```diff
@@ -0,0 +1,34 @@
+{
+    "version": "2021.1.0",
+    "description": "Tool for managing networks and network troubleshooting",
+    "homepage": "https://borntoberoot.net/NETworkManager/",
+    "license": "GPL-3.0-or-later",
+    "suggest": {
+        ".NET": "dotnet-sdk"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/BornToBeRoot/NETworkManager/releases/download/2021.1.0/NETworkManager_2021.1.0_Archiv.zip",
+            "hash": "425517a996ce52bb2adff05d02537817c954329402aae9f05773b36c61997e71"
+        }
+    },
+    "bin": "NETworkManager.exe",
+    "extract_dir": "NETworkManager",
+    "shortcuts": [
+        [
+            "NETworkManager.exe",
+            "NETworkManager"
+        ]
+    ],
+    "checkver": {
+        "url": "https://github.com/BornToBeRoot/NETworkManager/releases/latest",
+        "regex": "<a href=\"/BornToBeRoot/NETworkManager/releases/tag/([0-9\\.]+)\">\\1</a>"
+    },
+    "autoupdate": {
+        "url": "https://github.com/BornToBeRoot/NETworkManager/releases/download/$version/NETworkManager_$version_Archiv.zip",
```

Comment:
64bit

Reply:
Not sure what do you mean here.
Tried:
```patch
-        "url": "https://github.com/BornToBeRoot/NETworkManager/releases/download/$version/NETworkManager_$version_Archiv.zip",
+        "64bit": {
+            "url": "https://github.com/BornToBeRoot/NETworkManager/releases/download/$version/NETworkManager_$version_Archiv.zip"
+        },
```

, but it causes scoop to throw error:

`ERROR Exception calling ".ctor" with "1" argument(s): "Invalid URI: The URI is empty."`

Probably because `32bit` part is missing...

Reply:
https://github.com/lukesampson/scoop/blob/master/schema.json

Reply:
Fixed according to schema in bc6268e.